docs/GETTING_STARTED.md

```diff
@@ -6,12 +6,17 @@ The usage of `@daniloster/svelte-i18n` is based on some conventions which can sp
 
 ## Importing and using
 
-The library only exposes one function and the configuration has default values set in case they are not provided.
+The library only exposes 2 functions (`factoryI18nState` and `factoryI18nextState`) and the configuration has default values set in case they are not provided.
+
+Here we are starting development with a simple case, the proper lib is taking care of the internationalization. Basically, the look up for the text related to the json path.
 
 _`src/i18nState.js`_
 
 ```js
+// Either
 import factoryI18nState from '@daniloster/svelte-i18n'
+// or
+// import { factoryI18nState } from '@daniloster/svelte-i18n'
```

Comment:
this or seems like the code line before

Reply:
It is exported as default and named export. the diff is `{`

Reply:
ahh sorry, my eyes are tired

Reply:
No worries, thanks for the feedback though